cloudconfig/userdatacfg.go

```diff
@@ -41,6 +41,10 @@ type UserdataConfig interface {
 	// ConfigureJuju updates the provided cloudinit.Config with configuration
 	// to initialise a Juju machine agent.
 	ConfigureJuju() error
+	// ConfigureCustomOverrides updates the provided cloudinit.Config with
```

Comment:
we need a space before each commented attr in a struct

Reply:
updated all